Lib/test/test_uuid.py

```diff
@@ -600,7 +600,26 @@ def test_uuid1_time(self):
     def test_uuid3(self):
         equal = self.assertEqual
 
-        # Test some known version-3 UUIDs.
+        # Test some known version-3 UUIDs with name passed as a byte object
+        for u, v in [(self.uuid.uuid3(self.uuid.NAMESPACE_DNS,
+                        bytes('python.org', encoding='UTF-8')),
```

Comment:
```suggestion
                        b'python.org',
```
Just use a bytes literal (this also goes for the other examples here)

Reply:
I changed to bytes literals